src/dict.c

```diff
@@ -56,15 +137,10 @@
  * prevented: a hash table is still allowed to grow if the ratio between
  * the number of elements and the buckets > dict_force_resize_ratio. */
 static int dict_can_resize = 1;
-static unsigned int dict_force_resize_ratio = 5;
+/* static unsigned int dict_force_resize_ratio = 5; */
```

Comment:
I kept this for now, since there is a possibility to resize the root table (see the [paper](https://infoscience.epfl.ch/record/64398/files/idealhashtrees.pdf)). It adds to complexity but may improve speed. Not sure we needed it. If not, I'll delete this for sure.